pkg/plugin/plugin_test.go

```diff
@@ -350,6 +350,18 @@ func TestValidatePluginData(t *testing.T) {
 	}
 }
 
+func TestValidatePluginData_Empty_Plugin(t *testing.T) {
+	err := validatePluginData(nil, "")
```

Comment:
This can be one-lined: `if err := …; err != nil {`

You can assign a variable and evaluate it in the same line. This is used when you don’t want a variable name to live beyond the scope of the `if`, preventing code shadowing.